app/Models/Extensions/Thumb.php

```diff
@@ -81,11 +86,16 @@ public static function createFromQueryable(Relation|Builder $photoQueryable, Sor
 	 */
 	public static function createFromPhoto(?Photo $photo): ?Thumb
 	{
-		$thumb = $photo?->size_variants->getThumb();
+		$thumb = config('app.livewire', true) === true
+			? $photo?->size_variants->getSmall() ?? $photo?->size_variants->getThumb()
+			: $photo?->size_variants->getThumb();
 		if ($thumb === null) {
 			return null;
 		}
-		$thumb2x = $photo->size_variants->getThumb2x();
+
+		$thumb2x = config('app.livewire', true) === true
+			? $photo?->size_variants->getSmall2x() // We do note provide '?? sv->Thumb2x()' because if messes up on retina display
```

Comment:
I think it should actually be:

```suggestion
		$thumb2x = config('app.livewire', true) === true && $photo?->size_variants->getSmall()
			? $photo?->size_variants->getSmall2x()
```

No, `getSmall()` is not a typo -- it should repeat the same condition as when selecting `$thumb`; that way we ensure that `$thumb` and `$thumb2x` are of the same aspect ratio...

Reply:
Fixed [736d5ee](https://github.com/LycheeOrg/Lychee/pull/2138/commits/736d5ee064ca089c9b8cb3d05de30f8d8dc563be)